lib/govuk_app_config/govuk_error/configure.rb

```diff
@@ -5,7 +5,7 @@
     e
   }
 
-  config.silence_ready = !Rails.env.production? if defined?(Rails)
+  config.silence_ready = !Rails.env.production? if defined?("Rails")
```

Comment:
Did you mean to make Rails a string here?

Reply:
Oops - good spot!